test/app/src/testApp.ts

```diff
@@ -274,9 +300,10 @@ class TestApp {
     widgetConfig.authParams.state = state; // Must set authParams in constructor: OKTA-361428
 
     if (_forceRedirect) {
-      renderOptions.mode = 'remediation';
+      renderOptions.mode = 'remediation'; // since version 5.0
+      widgetConfig.authParams.display = 'page'; // version < 5.0
     } else {
-      widgetConfig.authParams.display = 'none';
+      widgetConfig.authParams.display = 'none'; // pversion < 5.0
```

Comment:
is the "p" in "pversion" significant, or a typo?

Reply:
typeo